### DISCUSSION TITLE
Recreate existing release

### DISCUSSION THREAD

**Discussion by Walkyst** [ncipollo/release-action#58] [Q&A]
It is possible with this action create release and if release with same tag already exist it will be recreated?

My main goal here is that always have fresh release with same tag `latestDevelopment`

**Comment by ncipollo** [comment]
Hi @Walkyst,
I think you would want to have a shell script as part of your workflow which updates your `latestDevelopment` tag, then run the release action against that tag. The script could do something like this:
```bash
git checkout <whatever-branch>
git pull
git tag -f latestDevelopment
git push -f origin latestDevelopment
```
